### DISCUSSION TITLE
Easylist CHN is not in Filter List > Languages, only Adguard Chinese available, why?

### DISCUSSION THREAD

**Discussion by Retia-Adolf** [dhowe/AdNauseam#2323] [Q&A] [answered]
As titled. I found this recently when I realized many of Chinese ADs are not blocked, the only default Chinese filter list "Adguard Chinese" doesn't seem as effective as before. But Easylist CHN seems not longer one of default lists in my AdNauseam setup, is this true for others?

**Comment by mneunomne** [comment, marked as answer]
@Retia-Adolf, sorry I have missed this until now. We follow the pattern of uBlock choices for the default filter lists. Here is the information I have found about it:

https://github.com/gorhill/uBlock/commit/de1ed2c519a9987411199db831fdd0ce38fd48c1

```
Replace "EasyList China" with "AdGuard Chinese"
As discussed internally with list maintainers:

- EasyList China doesn't leverage uBO's extended
  filter syntax

- EasyList China's home page is no longer available
  to the public

Additionally, "CJX's EasyList Lite" has been removed
from stock lists, as the list hosted on GitHub is no
longer updated, and "AdGuard Chinese" is the official
list to enable to address all filter issues for
Chinese sites.
```
